.rubocop.yml

```diff
@@ -5,4 +5,6 @@ AllCops:
     - bin/**/*
     - tmp/**/*
     - vendor/**/*
+    - gemfiles/vendor/**/*
+    - gemfiles/bin/**/*
```

Comment:
VERY little content: collapse it!
https://github.com/bundler/gemstash/blob/master/gemfiles/Gemfile.ruby-head